source/index.ts

```diff
@@ -296,30 +303,34 @@ is.nodeStream = (value: unknown): value is NodeStream => !is.nullOrUndefined(val
 
 is.infinite = (value: unknown) => value === Infinity || value === -Infinity;
 
-const isAbsoluteMod2 = (rem: number) => (value: number) => is.integer(value) && Math.abs(value % 2) === rem;
+const isAbsoluteMod2 = (rem: number) => (value: number): value is number => is.integer(value) && Math.abs(value % 2) === rem;
 is.even = isAbsoluteMod2(0);
 is.odd = isAbsoluteMod2(1);
 
-const isWhiteSpaceString = (value: unknown) => is.string(value) && /\S/.test(value) === false;
+is.emptyArray = (value: unknown): value is never[] => is.array(value) && value.length === 0;
+is.nonEmptyArray = (value: unknown): value is unknown[] => is.array(value) && value.length > 0;
 
-is.emptyArray = (value: unknown) => is.array(value) && value.length === 0;
-is.nonEmptyArray = (value: unknown) => is.array(value) && value.length > 0;
+is.emptyString = (value: unknown): value is string => is.string(value) && value.length === 0;
```

Comment:
we can even go for

```ts
value is ""
```

here :+1:

Reply:
That is very clever! Didn't realize that would work.